recipes/pyfftw/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "0.10.1" %}
+
+package:
+    name: pyfftw
+    version: {{ version }}
+
+source:
+    fn: pyFFTW-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/p/pyFFTW/pyFFTW-{{ version }}.tar.gz
+    md5: dab57de9d23f3b333115265914f3d240
+
+requirements:
+    build:
+        - cython
+        - python
+        - numpy x.x
+        - scipy      # For testing.
```

Comment:
For now I have dropped scipy. The unittests aren't being run in this iteration. Though the import tests are.